packages/layout/src/ChartPanel.ts

```diff
@@ -232,3 +233,16 @@ export class ChartPanel extends Border2 implements IHighlight {
     }
 }
 ChartPanel.prototype._class += " layout_ChartPanel";
+
+export interface ChartPanel {
+    highlight(): boolean;
+    highlight(_: boolean): this;
+    highlightSize(): number;
+    highlightSize(_: number): this;
+    highlightColor(): string;
+    highlightColor(_: string): this;
+}
+
+ChartPanel.prototype.publish("highlight", false, "boolean");
```

Comment:
`highlight` should not be published - so either mark it as private/internal or create a regular function.